HTTP.cabal

```diff
@@ -1,9 +1,9 @@
-Cabal-Version: >= 1.10
+Cabal-Version: 3.0
 Name: HTTP
 Version: 4000.4.1
 x-revision: 5
 Build-type: Simple
-License: BSD3
+License: BSD-3-Clause
```

Comment:
is it necessary to do this in the same PR?

Reply:
@Kleidukos @hsenag  I do not think it is good to require `cabal-version: 3.0` on the one hand (Cabal 3.0 was shipped with GHC 8.8) and `base >= 4.6` (GHC 7.6) on the other hand.  
Why not do the minimal bump on cabal-version and sync it with a base bump?  E.g. `cabal-version: 2.0` and `base >= 4.10`, setting GHC 8.2 as the last supported GHC version.
Ideally, `cabal-version` should be independent from the `Cabal` version, but it is not in practice.